### ISSUE TITLE
is asset files supported?

### ISSUE THREAD

**Issue by zozobreakzou** [pmq20/node-compiler#72]
if the node app use a resource file(such as a png or a csv file) with fs, does the nodec bundle them and the output exe can work on an new machine without these resource files.

**Comment by pmq20** [comment]
Yes

**Comment by zozobreakzou** [comment]
but failed in my test.  even the read_file1 function.
[node_app.zip](https://github.com/pmq20/node-compiler/files/1210742/node_app.zip)

**Comment by zozobreakzou** [comment]
in my opinion, it seems nodec bundle all the files in the 'cwd' in to '/__enclose_io_memfs__', is that right?
if so,  how to access the files in the bundle?

**Comment by pmq20** [comment]
That is correct. You could access it like this: `fs.readdirSync('/__enclose_io_memfs__')`.
